org-roam-capture.el

```diff
@@ -390,11 +426,18 @@ INFO is a plist for filling up Org-roam's capture templates.
 NODE is an `org-roam-node' construct containing information about the node.
 PROPS is a plist containing additional Org-roam properties for each template.
 TEMPLATES is a list of org-roam templates."
-  (let* ((props (plist-put props :call-location (point-marker)))
+  (let* ((props (thread-first props
+                  (plist-put :call-location (point-marker))
+                  (plist-put :action  (or (plist-get props :action) (if goto 'goto 'capture)))
+                  (plist-put :kind (or (plist-get props :kind) 'normal))))
```

Comment:
/s/kind/context
/s/normal/default

Reply:
It used to be named "context" (before I've pushed it upstream), but there's couple of problems with this name:
1. If you think about it, both `:if-new`, `:id`, `:action` (probably even more) already sort of specifying a more global context for capture, so the word is too generic. So if you would ask someone in an issue "In which capture *context* this template doesn't work?" it would be a bit of ambiguous. 
2. "Context" already used in [org-capture](https://code.orgmode.org/bzg/org-mode/src/3664447eb2417b38f0053674f248d2b0f1592ff0/lisp/org-capture.el#L553) and implies a different thing.

I've also tried "type", but again, it already implies different thing in both, org-roam-capture and org-capture. Since "kind" is not occupied by the both, I've rolled with it. Do you think I should I still s/kind/context?

Reply:
I do think `:kind` sounds a bit strange, and `:context` is much more appropriate (in the programming sense), but overloading terms with org-capture is definitely a bad idea. I was thinking something more descriptive would make sense like `:valid-if`, and then it could support more than just what `:kind` would (e.g. support arbitrary predicates).